Tower Forge -> Batch Forge (#14)
seqeralabs/docs: platform_versioned_docs/version-22.4.0/compute-envs/azure-batch.mdx

There are two ways to create a **Compute Environment** for **Azure Batch** with Tower.

1. **Batch Forge**: This option automatically manages the Azure Batch resources in your Azure account.

2. **Manual**: This option allows you to create a compute environment using existing Azure Batch resources.

> If you don't yet have an Azure Batch environment fully set up, it is suggested that you follow the [Batch Forge](#tower-forge) guide.

Reviewer: ```suggestion
If you don't yet have an Azure Batch environment fully set up, follow the [Batch Forge](#tower-forge) guide to do so.
```
